phys/module_ra_rrtmg_lw.F

```diff
@@ -2085,10 +2085,13 @@ module mcica_subcol_gen_lw
 !------------------------------------------------------------------
 ! Public subroutines
 !------------------------------------------------------------------
-
-      subroutine mcica_subcol_lw(iplon, ncol, nlay, icld, permuteseed, irng, play, hgt, &
-                       cldfrac, ciwp, clwp, cswp, rei, rel, res, tauc, cldfmcl, &
-                       ciwpmcl, clwpmcl, cswpmcl, reicmcl, relqmcl, resnmcl, taucmcl)
+! mji - Add height needed for exponential and exponential-random cloud overlap methods 
+!       (icld=4 and 5, respectively) along with idcor, juldat and lat used to specify
+!       the decorrelation length for these methods
```

Comment:
There are no other comments about other options and functionalities at the top of the routine call. Is this where this info should be added?

Reply:
I am withdrawing the question. The mcica_subcol_sw already has this type of comment, this is just being consistent.